pastepwn/analyzers/tests/emailpasswordpairanalyzer_test.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+import unittest
+from unittest import mock
+
+from pastepwn.analyzers.emailpasswordpairanalyzer import EmailPasswordPairAnalyzer
+
+
+class TestWordAnalyzer(unittest.TestCase):
+    def setUp(self):
+        self.obj = mock.Mock()
+
+    def test_match(self):
+        analyzer = EmailPasswordPairAnalyzer(None)
+        self.obj.body = "This is a Test"
+        self.assertFalse(analyzer.match(self.obj))
+
+        analyzer = EmailPasswordPairAnalyzer(None)
+        self.obj.body = "estocanam2@gmail.com:Firebird1@"
```

Comment:
Optional, but I would be 200% happy if you could add at least 5 positive and 5 negative tests.

Reply:
on it